indico/web/client/js/react/components/UserMenu.jsx

```diff
@@ -0,0 +1,111 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* eslint "react/forbid-component-props": "off" */
+
+import React from 'react';
+import propTypes from 'prop-types';
+import {Avatar, Divider, Icon, Select} from 'antd';
+
+
+window.build_url = (data) => JSON.stringify(data);
+
+import userDashboard from 'indico-url:users.user_dashboard';
+import userPreferences from 'indico-url:users.user_preferences';
+import authLogout from 'indico-url:auth.logout';
+import changeLanguage from 'indico-url:core.change_lang';
+
+import {Slot, ContainerDiv, ContainerBound} from 'indico/react/util';
+import {indicoAxios, handleAxiosError} from 'indico/utils/axios';
+
+import ArrowDownMenu from './ArrowDownMenu';
+import './UserMenu.module.scss';
+
+
+async function postAndReload(selectedLanguage) {
+    await indicoAxios.post(changeLanguage(), {lang: selectedLanguage});
+    location.reload();
+}
+
+export default function UserMenu({userData, staticData}) {
+    if (!userData) {
+        return '';
+    }
+
+    const {availableLanguages} = staticData;
+    const {firstName, lastName, email, language, isAdmin, avatarBgColor} = userData;
+    const avatar = (
+        <Avatar style={{
```

Comment:
Is this line break here intended? Would look nice in one line too.

Reply:
It's actually enforced by eslint.